frontend/src/app/rating/steps/applicant-score/applicant-score.component.html

```diff
@@ -0,0 +1,65 @@
+<h1>Applicant Rating</h1>
+
+<ion-list lines="none">
+  <ion-list-header>
+    Rhetoric:
+    {{ ratingForm.controls.applicantScore.controls.rhetoric.value }}
+  </ion-list-header>
+  <ion-item class="rating">
```

Comment:
May be we can implement this as an array of objects. And use ngFor to iterate in the template. But need to make sure two way binding happens when value changes. The current way of doing will become cumbersome if we add more list.

Reply:
I think this is something we could refactor later... We need to think about how to implement various other things (like routing and stuff like that), so we could refactor it then I guess.